### PR TITLE
feat: Group attendance display encounter

### LINKED ISSUE #7127: Show group attendance in encounter summary
### Describe the problem
When attendance is taken for the group session, the attendance is not displayed on the encounter summary page. 

### Describe the solution you'd like

Showing the group attendance on the encounter summary page would be advantageous.

### PR DESCRIPTION
**Pull request by juggernautsei**
Fixes #7127

#### Short description of what this resolves:
The attendance for a group meeting is not displayed on the encounter summary page.

#### Changes proposed in this pull request:
This adds the display of the group attendance to the encounter summary page.

### CODE REVIEW

File: interface/forms/group_attendance/report.php [hunk 1]
```diff
@@ -38,6 +41,35 @@ function group_attendance_report($pid, $encounter, $cols, $id)
...
+            </tr>
+            <?php
+            if ($participants) {
+                foreach ($participants as $participant) {
+                    $name = $participant['lname'] . ', ' . $participant['fname'];
+                    $attnStatus = getAttendanceStatus($participant['meeting_patient_status'])
+                    ?>
+                    <tr>
+                        <td><span class='text'><?php echo text($name); ?></span></td>
+                        <td><span class='text'><?php echo text($attnStatus); ?></span></td>
```
Comment (bradymiller) [inline interface/forms/group_attendance/report.php:58]: This is from a list, so should translate with the xl_list_label() function:
```
<td><span class='text'><?php echo text(xl_list_label($attnStatus)); ?></span></td>
```

Comment (juggernautsei) [reply interface/forms/group_attendance/report.php]: Bump..

**Comment by juggernautsei** [comment]
Bump in the afternoon and not in the night. This needs to make the patch.

**Review by stephenwaite — APPROVED** [review]

**Comment by adunsulag** [comment]
Pulled this down locally and tested.  Looks good.

**Pull request merged**